examples/minimal/README.md

```diff
@@ -15,6 +15,8 @@ No requirements.
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
 | ami\_id | AMI to use for Tamr EC2 instance | `string` | n/a | yes |
+| egress\_protocol | Protocol for egress rules. If not icmp, icmpv6, tcp, udp, or all use the protocol number. | `string` | n/a | yes |
```

Comment:
This isn't the in the variables.tf file, you may need to run make terraform/docs again.

Reply:
Removed unneeded variable.